modules/agar/src/main/ts/ephox/agar/api/TestStore.ts

```diff
@@ -55,6 +58,8 @@ export const TestStore = (): TestStore => {
 
   const assertEq = (label: string, expected: any[]) => Assert.eq(label, expected, array.slice(0));
 
+  const assertSortedEq = (label: string, expected: any[]) => Assert.eq(label, expected.slice(0).sort(), array.slice(0).sort());
+
   const sAssertSortedEq = (label: string, expected: any[]) => Step.sync(() =>
   // Can't use a normal step here, because we don't need to get array lazily
     Assert.eq(label, expected.slice(0).sort(), array.slice(0).sort())
```

Comment:
This also should not have duplicate code. You now have 2 implementations for `assertSortedEq` and they should be using the same implementation.

- [ ] Make `sAssertSortedEq` call `assertSortedEq` to remove duplication

Reply:
Removed duplicated call in `sAssertSortedEq`, `sAssertSortedEq` is now calling `assertSortedEq`